assessments/projects/opa/README.md

```diff
@@ -0,0 +1,111 @@
+
+**Analysis**: There is a clear benefit for adopters who have heterogeneous
+infrastructure or high rate of change where lack of policy enforcement would
+present significant business risk. The added complexity of OPA is not trivial.
+OPA enables “policy as code” and Rego policy expressions require the same care
+to develop as any security critical code. OPA reduces risk by isolating policy
+from business logic. Concise declarative policy has the potential to support
+efficient review; however, that relies upon a reviewer who fully understands the
+syntax and tools for effective validation. Hence usability and security around
+Rego and interaction between aspects of OPA becomes more critical.
+
+All questions from reviewers were addressed in [self-assessment](self-assessment.md)
+with non-critical issues captured as issues and noted below.
+
+## Recommendations
+
+Rego has a non-trivial learning curve and even in demos shows there is potential
+for confusion. Examining the language’s potential for confusion and mitigating
+situations where these problems arise would help to improve user security.
+
+### Project recommendations
+
+1. ([OPA-1699](https://github.com/open-policy-agent/opa/issues/1699)) System
+design and deployment guidance to include “gotchas” laying out potential issues
+in a way that is more understandable to users.
+   
+    a. Common scenarios or design challenges, such as policy loading to handled
+	   race/error conditions, initialization, and queries that require state
+
+    b. Deployment Checklist: these so that one can check more holistically that
+	   they are using OPA correctly before using it in prod.
+
+2. ([OPA-1700](https://github.com/open-policy-agent/opa/issues/1700)) Rego usability
+
+    a. Make it more difficult for users to make policy-related errors by improving
+	   OPA toolchain usability (testing, playground). Related Issues: [OPA-1697](https://github.com/open-policy-agent/opa/issues/1697)
+	   and [OPA-1698](https://github.com/open-policy-agent/opa/issues/1698)
+
+    b. Make it easier for integrators to understand the security ramifications of
+	   different OPA deployment models, including models where the RESTful API
+	   is used to manipulate policies
+
+3. Expand security response team to include participants outside of Styra
+
+4. Work toward better security by improving OPA tooling and Rego language
+
+    a. OPA decision logs may contain sensitive information by default e.g.
+	when used as Kubernetes Admission Controller depending on the configuration
+	also secrets could be part of the decision logs.  While [OPA configuration](https://www.openpolicyagent.org/docs/latest/management/#masking-sensitive-data)
+	allows specific data to be omitted, we recommend that OPA create mechanisms
+	to make it harder for users to accidentally leak secrets using OPA.
+	Related Issues: [OPA-1781](https://github.com/open-policy-agent/opa/issues/1781)
+
+
+### CNCF recommendations
```

Comment:
suggestion from @dshaw @pragashj -- for clarification, let's rename:  "Recommendations to CNCF"

Reply:
Done